hip-0000-template.md

```diff
@@ -1,59 +1,276 @@
+To maintain the highest level of security and data integrity, it is strongly recommended to avoid using IPNS for DEA metadata. Instead, adhere to the principles of immutability provided by IPFS, or protocols built on top, itself. This ensures that once data is recorded and associated with an asset, it remains unaltered and reliable throughout its lifecycle.
 
 ## How to Teach This
 
-For a HIP that adds new functionality or changes interface behaviors, it is helpful to include a section on how to teach users, new and experienced, how to apply the HIP to their work.
+Guardian-compliant HTS implementations use this standard to provide additional metadata for their DEAs.
+
+Wallet and token explorer implementations interrogate HTS tokens using this standard to display additional metadata for tokens.
 
 ## Reference Implementation
 
-The reference implementation must be complete before any HIP is given the status of “Final”. The final implementation must include test code and documentation.
+This section provides examples of a metadata schema adhering to this specification.
+
+### Example Schema: Required and Recommended fields ONLY
+
+```json
+{
+    "name": "Redhill Farm Carbon Credit",
+    "creator": "Redhill Farm",
+    "format": "guardian-dea",
+    "description": "This is a carbon credit issued by Redhill Farm",
+    "image": "ipfs://bafybeiapq5ewrpxrsche54laxvwwvth2ye3df4wbswwe3z2mk7rgzlz7za",
+    "type": "image/png",
+    "properties": {
+        "guardian_proof": {
+            "type": "Ed25519Signature2018",
+            "created": "2023-10-04T11:18:59Z",
+            "verificationMethod": "did:hedera:testnet:Ci3QBHbe3Fa6gKn8hm44PJn1uMkXEDbJkPQboYN7EKWV_0.0.3160930#did-root-key",
+            "proofPurpose": "assertionMethod",
+            "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..c3Jmwg3Nis5puQLNh5zcMgkzmmsB-m10A9dOoGh_3tt8aaTIzMZSsxQJbiLmb795mw-sKfa9DqN7gy3kbgNDCA"
+        }
+    },
+    "attributes": [
+        {
+            "trait_type": "timestamp",
+            "value": 1710271999.591571
+        }
+    ]
+}
+```
+
+### Example Schema: Full-featured
 
+```json
+{
+    "name": "Redhill Farm Carbon Credit",
+    "creator": "Redhill Farm",
+    "format": "guardian-dea",
+    "description": "This is a carbon credit issued by Redhill Farm",
+    "image": "ipfs://bafybeiapq5ewrpxrsche54laxvwwvth2ye3df4wbswwe3z2mk7rgzlz7za",
+    "type": "image/png",
+    "attributes": [
+        {
+            "trait_type": "timestamp",
+            "value": "1710271999.591571003"
```

Comment:
The default display type in HIP412 is `text`. We can help explorers by adding the way we would like this represented.

```suggestion
            "trait_type": "timestamp",
            "display_type": "datetime",
            "value": "1710271999.591571003"
```

Reply:
Agreed, is there a way for a attribute to be non-indexable as filtering on timestamps is nonsense